lib/3scale_toolbox/commands/backend_command/copy_command.rb

```diff
@@ -0,0 +1,64 @@
+require '3scale_toolbox/commands/backend_command/copy_command/task'
+require '3scale_toolbox/commands/backend_command/copy_command/create_or_update_target_backend_task'
+require '3scale_toolbox/commands/backend_command/copy_command/copy_metrics_task'
+require '3scale_toolbox/commands/backend_command/copy_command/copy_methods_task'
+require '3scale_toolbox/commands/backend_command/copy_command/copy_mapping_rules_task'
+
+module ThreeScaleToolbox
+  module Commands
+    module BackendCommand
+      class CopySubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'copy'
+            usage       'copy [opts] -s <source_remote> -d <target_remote> <source_backend>'
+            summary     'Copy backend'
+            description <<-HEREDOC
+            This command makes a copy of the referenced backend.
+            Target backend will be searched by source backend system name. System name can be overriden with `--target_system_name` option.
```

Comment:
Can we explain the merge behaviour in case the destination backend already exists?
What happens with the attributes of the destination backend? and with its metrics, methods and mapping rules whether they exist or not?